db/config.cc

```diff
@@ -691,6 +691,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , cpu_scheduler(this, "cpu_scheduler", value_status::Used, true, "Enable cpu scheduling")
     , view_building(this, "view_building", value_status::Used, true, "Enable view building; should only be set to false when the node is experience issues due to view building")
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Used, true, "Enable SSTables 'mc' format to be used as the default file format")
+    , enable_cdc(this, "enable_cdc", value_status::Used, false, "Enable Change Data Capture")
```

Comment:
Why do we need this flag? There is already a per-table flag.

Reply:
We need this to be able to do rolling upgrade of the new Scylla binary with CDC schema changes not present and then enable CDC on such cluster. It would work like this:
1. Do rolling upgrade with enable-cdc = false
2. Switch enable-cdc to true on all nodes
Otherwise we will get the problem described in #5288

Reply:
No need for a flag. Condition everything on cluster_supports_cdc() (which is only true if all nodes report they support cdc, and each node only enables the feature if it is in experimental mode).